### PR TITLE
chore(functional_tests): update CI image to add different browsers

### PR DESCRIPTION
**Pull request by ashrivastava-qa**
## Because

- To be able achieve Browser compatibility and run tests in multiple browsers we have to update the CI runner image with the new browsers installed.

## This pull request

- Updates the CI runner image with Firefox, Chromium and webkit installed.

## Issue that this pull request solves

Closes: FXA-10306

## Checklist

_Put an `x` in the boxes that apply_

- [x] My commit is GPG signed.
- [ ] If applicable, I have modified or added tests which pass locally.
- [ ] I have added necessary documentation (if appropriate).
- [ ] I have verified that my changes render correctly in RTL (if appropriate).

## Screenshots (Optional)

Please attach the screenshots of the changes made in case of change in user interface.

## Other information (Optional)

Any other information that is important to this pull request.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -978,6 +978,7 @@ workflows:
                 - main
                 - chore/update-tsconfig-for-node-20
                 - update-functional-test-executor
+                - update-ci-image
```
Comment (Trinaa) [inline .circleci/config.yml:981]: This is interesting. When do these branch filters get cleaned-up?

File: _dev/docker/ci/Dockerfile [hunk 2]
```diff
@@ -33,7 +32,9 @@ RUN yarn install --immutable;
 # must based on cimg/node:20.11-browsers, which is why this stage
 # is necessary.
 FROM builder as playwright-install
-RUN npx playwright install firefox;
+RUN npx playwright install --with-deps firefox \
+ && npx playwright install --with-deps chromium \
+ && npx playwright install --with-deps webkit;
```
Comment (Trinaa) [inline _dev/docker/ci/Dockerfile:37]: This seems verbose and i wonder if a playwright installation is attempted multiple times. Would something like the following work? 🤔 
```suggestion
RUN npx playwright install --with-deps firefox chromium webkit;
```

The [documentation](https://playwright.dev/docs/browsers#install-system-dependencies) also makes me wonder if '--with-deps' can be used standalone?

Comment (ashrivastava-qa) [reply _dev/docker/ci/Dockerfile:37]: Yes you are right. I will change it.

File: .circleci/config.yml (hunk 1 above)
Comment (ashrivastava-qa) [reply .circleci/config.yml:981]: Good question. I am not sure as the [ecosystem doc](https://mozilla.github.io/ecosystem-platform/how-tos/ci-guidelines#updating-ci-runner-images) doesn't mentions anything about removing but just adding which is what I was following to update the CI runner. But IMO we can remove the old branches after the patch lands, I think.

Comment (ashrivastava-qa) [reply .circleci/config.yml:981]: I will confirm on this and remove the old branches.

**Review by Trinaa — APPROVED** [review]
Awesome! 🚀 
Reminder, just in case, to squash format the commit message according to [CONTRIBUTING.md](https://github.com/mozilla/fxa/blob/main/CONTRIBUTING.md#body)

File: .circleci/config.yml (hunk 1 above)
Comment (ashrivastava-qa) [reply .circleci/config.yml:979]: The old branches have been removed.

**Pull request merged**